### PR TITLE
Update serverless functions

### PR DESCRIPTION
**Pull request by yasakova-anastasia**
### Motivation and context

Updated serverless functions: f-BRS, HRNET, IOG, Semantic segmentation for ADAS, text detection v4, Mask RCNN via Tensorflow

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added a description of my changes into [CHANGELOG](https://github.com/cvat-ai/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/cvat-ai/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/cvat-ai/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/cvat-ai/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/cvat-ai/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/cvat-ai/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/cvat-ai/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

**Comment by azhavoro** [comment]
@yasakova-anastasia Hi,
I got these errors:

IOG:
iog

semantic_segmentation_adas:
semantic_segmentation_adas

text_detection:
text_detection

Used version:
commit d9b8b0314d48faa20bb7a9f046f8fd73609ed3f3

**Comment by yasakova-anastasia** [comment]
@azhavoro, I fixed these issues.

**Comment by bsekachev** [comment]
Maybe this not related with the PR, but I've noticed the following.
For ``Semantic Segmentation ADAS`` detector:

The mask it returns for sidewalk is:
image

But the polygon is:
image

**Comment by bsekachev** [comment]
After I run Mask RCNN offline detector, I get this error when try to open a job:
image

It is because the server returns objects which are ``masks``, but their points looks like polygon points. 

For now we do not have an option like ``Convert masks to polygons``, when run detectors offline. 
I will try to add it ASAP..

File: serverless/openvino/omz/intel/semantic-segmentation-adas-0001/nuclio/model_handler.py [hunk 1]
```diff
@@ -8,6 +9,14 @@
 from skimage.measure import approximate_polygon, find_contours
 from model_loader import ModelLoader
 
+
+def to_cvat_mask(box: list, mask):
+    xtl, ytl, xbr, ybr = box
+    flattened = mask[ytl:ybr + 1, xtl:xbr + 1].flat[:].tolist()
+    flattened.extend([xtl, ytl, xbr, ybr])
+    return flattened
```
Comment (bsekachev) [inline serverless/openvino/omz/intel/semantic-segmentation-adas-0001/nuclio/model_handler.py:17]: Would it be an option, to put shared code to ``serverless/common/shared.py`` or something like this?

**Comment by bsekachev** [comment]
Issue with offline running looks fixed. Could you look at Pylint issues?

@kirill-sizov Do you have any ideas why we have Remark linter failed here if therea re not any .md files touched?

**Comment by bsekachev** [comment]
@kirill-sizov 

> The error occurs because the branch bs/masks into which this PR is merging contains this error. Need to fix CHANELOG.md in bs/masks

I know that ``bs/masks`` contains this error, but it is not clear why I see it in this PR where this file was not changed :)

File: serverless/openvino/omz/intel/semantic-segmentation-adas-0001/nuclio/model_handler.py (hunk 1 above)
Comment (yasakova-anastasia) [reply serverless/openvino/omz/intel/semantic-segmentation-adas-0001/nuclio/model_handler.py]: Moved `to_cvat_mask` to a separate file

**Comment by yasakova-anastasia** [comment]
> Could you look at Pylint issues?

The issues with Pylint are related to how `ModelLoader` is imported. If we fix this, there will be an error when deploying the function.

**Comment by bsekachev** [comment]
@yasakova-anastasia 

If some issues are expected, we can at lease disable checks for specific lines.

**Review by bsekachev — APPROVED** [review]

**Pull request merged**